docs/specification.md

```diff
@@ -873,6 +873,8 @@ Parameter (in Michelson):
 This contract implements [TZIP-016](https://gitlab.com/tzip/tzip/-/blob/21fb73fe01df8a744c9b03303e3d73b0f2265eb2/proposals/tzip-16/tzip-16.md).
 
 The DAO contract itself won't store the metadata, rather a dedicated contract will contain that.
+The metadata can also be uploaded to IPFS file storage.
+
 Motivation:
 * The baseDAO contract is very large and already over the Tezos hard limits.
```

Comment:
We forgot to update this, it seems, the contract is no longer over the limit.

Reply:
I've removed that part.